### PR TITLE
add: new resources reserved on customers nodes

### PR DESCRIPTION
**Pull request by SebastienJardin**

### CODE REVIEW

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md [hunk 1]
```diff
@@ -42,15 +42,13 @@ The OS and Docker demon version on your nodes will be regularly updated. Current
 * Docker: 18.06.3
 * Containerd: 1.4.3
 
-
```
Comment (ricoleabricot) [inline pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md:45]: why removing the double return line ? It's interpreted by markdown to make a bigger space between blocks

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md [hunk 2]
```diff
@@ -94,5 +92,35 @@ Feature gates:
 
 ## Reserved resources
 
-Each worker node has 1 GB of RAM and 100 mCPU reserved for Kubernetes components.  
+Each worker node has GB of RAM and mCPU reserved for Kubernetes components.
```
Comment (ricoleabricot) [inline pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md]: You forget the numbers ?
If you want to say it's variable, say something like: a certain amount of

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md [hunk 3]
```diff
@@ -94,5 +92,35 @@ Feature gates:
 
 ## Reserved resources
 
-Each worker node has 1 GB of RAM and 100 mCPU reserved for Kubernetes components.  
+Each worker node has GB of RAM and mCPU reserved for Kubernetes components.  
 This reserved quotas may evolve in the future, the page will be updated when it does
+
+To guarantee the availabilty of customers node, we reserverd resources (CPU and RAM), proportionally on the instance flavor.
```
Comment (ricoleabricot) [inline pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md]: s/reserverd/are reserving

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md [hunk 4]
```diff
@@ -94,5 +92,35 @@ Feature gates:
...
+
+* CPU
+
+The calcul used to estimed the reservation is :  (10 % of 1 CPU + 0,5% of all CPU cores)
```
Comment (ricoleabricot) [inline pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md]: s/estimed/estimate

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md [hunk 5]
```diff
@@ -94,5 +92,35 @@ Feature gates:
...
+The unit is in seconds: 1 CPU = 1 second, 10% of 1 CPU = 100 ms, 50% of 2 CPU = 1 s
+
+For example for a flavor b2-15, with 4 cpu: (10 % of 1 CPU + (0,5 * 4)= 120 ms
+
+* RAM
+
+The calcul used to estimed the reservation is : ( (768M Kubernetes + 5% of total memory) + ( 256M system) )
```
Comment (ricoleabricot) [inline pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md]: s/estimed/estimate

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md [hunk 6]
```diff
@@ -94,5 +92,35 @@ Feature gates:
...
+
+* Storage
+
+Mathematical "log" function is egal to "ln(10)".
+
+The calcul used to estimed the reservation is : ( ( log(total storage)*2 ) + ( log(total storage)*8 ) + ( 10% of total storage ) )
```
Comment (ricoleabricot) [inline pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md]: s/estimed/estimate

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md [hunk 7]
```diff
@@ -94,5 +92,35 @@ Feature gates:
...
+for example: `log(100) = 2`.
+
+This array helping you to predict resouces available.
```
Comment (ricoleabricot) [inline pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md]: s/helping/helps

Comment (ricoleabricot) [reply pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md]: s/resouces available/available resources

**Review by ricoleabricot — CHANGES_REQUESTED** [review]

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md (hunk 1 above)
Comment (SebastienJardin) [reply pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md:45]: My markdown linter was not happy :/

File: pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md (hunk 2 above)
Comment (SebastienJardin) [reply pages/platform/kubernetes-k8s/exposed-apis-software-versions-reserved-resources/guide.en-us.md]: yes, it is variable depending on flavor

**Review by ricoleabricot — APPROVED** [review]

**Review by bs-ovh — APPROVED** [review]

**Comment by bs-ovh** [comment]
Thank you for your contribution!

**Pull request merged**